packages/tocco-ui/src/util/htmlMarkup.js

```diff
@@ -0,0 +1,52 @@
+export const htmlMarkup = `
```

Comment:
this file exists is solely for the story? Please name it like that. 
or better... why dont you just move it into the story.

Reply:
I moved it to a separate file because I had the assumption that code should only be written once and then be referenced. I use this markup in two files.